### PR TITLE
1575 update login with orcid screen with new designs

### LINKED ISSUE #1575: Update green 'Login with Orcid' screen
## Problem / Motivation
- WHO wants this? Authors directed to xPub
- WHEN and WHERE do they want it? As the first screen they see, IF they have not already logged in via the eLife profiles service
- WHAT do they want? A screen that outlines the journal's vision and invites the user to log in using their ORCiD credentials
- WHY do they want this? So they are aware of the scope of the journal before they attempts to submit a manuscript for consideration

## Proposed solution
- Repurpose the current 'green' version of the login screen to include the updated text
- Add new grid layout
- Add new image asset
- Include internal and external links
- 'author guide' is an internal link to https://reviewer.elifesciences.org/author-guide
- 'Terms and conditions' is an external link to https://elifesciences.org/terms
- 'Privacy policy' is an external link to https://elifesciences.org/privacy

## Clarification needed and assumptions
- This screen will appear for any user who is not already logged in via eLife profiles service.

---

## Tasks

- [ ] A description of a task
- [ ] A description

## Technical notes
- We could use this technology
- More notes or suggestions

## User interface / Wireframes
screen shot 2019-02-25 at 12 29 56

### PR DESCRIPTION
**Pull request by will-byrne**
#### Background
There are new designs for login and redirect pages
image
This work is applying the new designs for login-updates-login-page

### Dev requirements
- [x] Create a new ImageWrapper atom to hold all the images
- [x] Add necessary updates to styleguide
- [x] Create FooterPrivicy molecule to be reused in all similar instances.
- [x] Investigate why assets are not working in styleguide

### Product requirements
- [x] Repurpose the current 'green' version of the login screen to include the updated text
- [ ] Add new grid layout
- [x] Add new image asset
- [x] Include internal and external links
- [x] 'author guide' is an internal link to https://reviewer.elifesciences.org/author-guide
- [x] 'Terms and conditions' is an external link to https://elifesciences.org/terms
- [x] 'Privacy policy' is an external link to https://elifesciences.org/privacy
- [x] Request review against designs

#### Any relevant tickets

Closes #1575

Design: https://app.zeplin.io/project/5bacf042615a8b6e20e84a5f/screen/5c742bfd30283132e874a726

### CODE REVIEW

**Comment by diversemix** [comment]
Hey! ... do you want to add to these to the milestone?  https://github.com/elifesciences/elife-xpub/milestone/12

This way you can get small changes into the code base faster :wink:

**Review by hdrury1 — APPROVED** [review]
lgtm

**Comment by Cooryd** [comment]
Just a heads up, this work might also require some e2e test changes before it can pass in the deployment pipeline https://alfred.elifesciences.org/blue/organizations/jenkins/test-elife-xpub/detail/test-elife-xpub/812/pipeline

File: app/components/pages/Login/index.js [hunk 1]
```diff
@@ -80,55 +90,69 @@ class LoginPage extends React.Component {
...
-          </Box>
-        ) : (
-          <Flex alignItems="center" mb={5}>
-            <Box mr={3}>
-              <a href={loginUrl}>
-                <ButtonOrcid data-test-id="login">Login with ORCID</ButtonOrcid>
-              </a>
-            </Box>
+      <Box
+        mx={['24px', '24px', '12.5%', '8.333%', '16.666%']}
```
Comment (hem-brahmbhatt) [inline app/components/pages/Login/index.js]: We can reference these values from the theme. You can find the theme [here](https://github.com/elifesciences/elife-xpub/blob/develop/client/elife-theme/src/index.js#L50-L62) and it's usage for spacing [here](https://github.com/elifesciences/elife-xpub/blob/develop/app/components/global/AppBar/AppBar.js#L14).

Comment (will-byrne) [reply app/components/pages/Login/index.js]: This was changed to be static at all breakpoints in the latest commit but will remember for future reference.

**Comment by hem-brahmbhatt** [comment]
Redirect page does not centre the content at a shorter breakpoint after the image disappears.

Screen Shot 2019-03-20 at 15 15 16

**Comment by will-byrne** [comment]
> Redirect page does not centre the content at a shorter breakpoint after the image disappears.
> 
> Screen Shot 2019-03-20 at 15 15 16

Talked with @hdrury1  and @diversemix  and this will be a separate piece of work that is currently de-prioritised.

**Review by diversemix — APPROVED** [review]

**Review by chris-huggins — APPROVED** [review]

**Pull request merged**